### PR TITLE
Add auto-updating for svg4everybody

### PR DESCRIPTION
**Pull request by JoshMock**
Added auto-updating for svg4everybody.

### CODE REVIEW

**Comment by JoshMock** [comment]
Pushed a change to fix the `basePath`. Should be good now.

File: files/svg4everybody/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "svg4everybody",
+  "repo": "jonathantneal/svg4everybody",
+  "files": {
+    "basePath": ".",
```
Comment (jimaek) [inline files/svg4everybody/update.json]: The correct is `/`

**Comment by JoshMock** [comment]
@jimaek Fixed!

**Comment by jimaek** [comment]
Looks good, thank you

**Pull request merged**